app/services/katello/cert/ca.rb

```diff
@@ -0,0 +1,21 @@
+module Katello
+  module Cert
+    module Ca
+      def self.ueber_cert(repository)
+        repository.organization.debug_cert
+      end
+
+      def self.ca_cert
+        File.open(Setting[:ssl_ca_file], 'r').read
+      end
+
+      def self.ssl_client_cert
+        OpenSSL::X509::Certificate.new(File.open(SETTINGS[:katello][:ssl_client_cert], 'r').read)
+      end
+
+      def self.ssl_client_key
+        OpenSSL::PKey::RSA.new(File.open(SETTINGS[:katello][:ssl_client_key], 'r').read)
+      end
```

Comment:
:+1: There is a mixture of things in here that don't align to what I think of with 'Ca' as the module name. A bit of bikeshedding but worth it.